kornia/augmentation/functional.py

```diff
@@ -6,7 +6,10 @@
 from kornia.geometry.transform.flips import hflip, vflip
 from kornia.geometry.transform import (
     get_perspective_transform, warp_perspective, center_crop, rotate, crop_by_boxes)
-from kornia.color.adjust import AdjustBrightness, AdjustContrast, AdjustSaturation, AdjustHue
+from kornia.color.adjust import (
+    # TODO: Remove module-like components
```

Comment:
From functional you mean?

Reply:
Yes. apply_colorjitter in this file creates a moduleList every time. It gonna be using pure functional functions.